### ISSUE TITLE
driver/block_dev.h: no such file or directory

### ISSUE THREAD

**Issue by zhouminglong** [embox/embox#2549]
make confload-project/pjsip/stm32f746g-discovery
make 

fatal error ：driver/block_dev.h: no such file or directory
include <driver/block_dev.h>

**Comment by anton-bondarev** [comment]
Thanks!
To fix the problem just add `include embox.fs.driver.devfs_stub` into 'conf/mods.conf' as in #2550
You can also get repo master
Don't remember to modify 'conf/simple_pjsua_sip_account.inc`
```
#define SIP_DOMAIN     "sip.linphone.org"
#define SIP_USER       "yyyyy"
#define SIP_PASSWD     "xxx"
```

**Comment by wawuwo** [comment]
Hi @anton-bondarev! Shouldn't this issue be closed? It look likes like you fixed it in #2550

**Comment by anton-bondarev** [comment]
> Hi @anton-bondarev! Shouldn't this issue be closed? It look likes like you fixed it in https://github.com/embox/embox/pull/2550

@wawuwo Thanks! I've closed it
